src/gnome-utils/gnc-gnome-utils.c

```diff
@@ -182,13 +182,20 @@ gnc_add_css_file (void)
 
     provider_user = gtk_css_provider_new ();
     provider_app = gtk_css_provider_new ();
+    provider_fallback = gtk_css_provider_new ();
     display = gdk_display_get_default ();
     screen = gdk_display_get_default_screen (display);
+
+    gtk_style_context_add_provider_for_screen (screen, GTK_STYLE_PROVIDER (provider_fallback), GTK_STYLE_PROVIDER_PRIORITY_THEME - 50);
```

Comment:
When I first tried this I thought it did not work, tested again and it does, will change.